### DISCUSSION TITLE
Question about the domain table

### DISCUSSION THREAD

**Discussion by ghost** [postfixadmin/postfixadmin#467] [Q&A]
Hi all. I have a question about the database (using PostgreSQL). Can someone answer the following questions I had on my mind (about the `domain` table)?
1. What are the `aliases`, `mailboxes`, `maxquota` and `quota` fields about?
2. How do you set a domain's quota to unlimited? Do you set the `quota` field to `0`?
3. What are the quotas measured in? MB?

Thanks, all help appreciated.

**Comment by ghost** [comment]
Also, how does the quota itself work? Does it have an interval that checks to see the storage sizes of each mailbox/domain?

**Reply by ghost** [reply]
Also, one more thing I wanted to ask. Does making a domain inactive cause all of its mailboxes to be inactive as well? Or would I have to make the mailboxes inactive separately?

**Comment by DavidGoodwin** [comment]
The easy ones : 

 * domain.aliases - max number of aliases a domain can have; set to 0 for unlmited.
 * domain.mailboxes - max number of mailboxes a domain can have; set to 0 for unlimited


The "I've not got much of an idea about these" ones : 

 * domain.quota - appears to become the 'max' quota allowable for a mailbox you're adding to the domain.
 * domain.maxquota - appears to correspond to the 'Mailbox Quota (MB)' field in the domain list page; but I'm not sure what this does?
 * mailbox.quota - should be the max quota available for a mailbox? 

I've never used quota with Postfix/Dovecot/Postfixadmin, so I have never paid much attention to them.
